tests/unittests/test_losses.py

```diff
@@ -137,3 +138,43 @@ def test_transducer_loss():
     )
     out_cost.backward()
     assert out_cost.item() == 2.247833251953125
+
+@parameterized.expand([("cpu"), ("cuda")])
+def test_pytorch_transducer_loss(device):
+    if device == "cuda" and torch.cuda.device_count() == 0:
+        pytest.skip("This test can only be run if a GPU is available")
+
+    from torchaudio.prototype.rnnt_loss import rnnt_loss
+
+    device = torch.device(device)
+    log_probs = (
+        torch.Tensor(
+            [
+                [
+                    [
+                        [0.1, 0.6, 0.1, 0.1, 0.1],
+                        [0.1, 0.1, 0.6, 0.1, 0.1],
+                        [0.1, 0.1, 0.2, 0.8, 0.1],
+                    ],
+                    [
+                        [0.1, 0.6, 0.1, 0.1, 0.1],
+                        [0.1, 0.1, 0.2, 0.1, 0.1],
+                        [0.7, 0.1, 0.2, 0.1, 0.1],
+                    ],
+                ]
+            ]
+        )
+        .to(device)
+        .requires_grad_()
```

Comment:
also removed `log_softmax` here (using `fused_log_softmax=True` default), but should not affect the loss